var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -83,21 +84,25 @@ def patch(self):
         makefile.filter('FC = ifort',       'FC ?= ifort', **kwargs)
         makefile.filter('FC = gfortran',    'FC ?= gfortran', **kwargs)
 
-    def manual_install(self, prefix):
+    def install(self, spec, prefix):
         spec = self.spec
         install_tree('include', prefix.include)
         if '~header-only' in spec:
             install_tree('lib', prefix.lib)
-        doc_path = prefix.share + '/libxsmm/doc'
+        doc_path = join_path(prefix.share, 'libxsmm', 'doc')
         mkdirp(doc_path)
-        for doc_file in glob('documentation/*.md'):
+        for doc_file in glob(join_path('documentation', '*.md')):
             install(doc_file, doc_path)
-        for doc_file in glob('documentation/*.pdf'):
+        for doc_file in glob(join_path('documentation', '*.pdf')):
             install(doc_file, doc_path)
-        install('README.md', doc_path)
-        install('LICENSE', doc_path)
-
-    def install(self, spec, prefix):
+        if '@1.8.2:' in spec:
+            install('LICENSE.md', doc_path)
+        else:
+            install('README.md', doc_path)
+            install('LICENSE', doc_path)
+        install('version.txt', doc_path)
+
+    def build(self, spec, prefix):
```

Comment:
Can you move this above install to keep things in order? Also, can you convert the below InstallError to a conflict?